### PR TITLE
Fix some issues with rmgpy.qm.qmdata

### PR DESCRIPTION
**Pull request by nickvandewiele**
This PR fixes issues with some of the qm classes:

- limit the no. of threads used by MOPAC calculations to 1 (to avoid CPU contention when concurrent thermo will be implemented)
- remove mutable argumets in QMData constructor
- remove CCLibData class and replace it by a module function. It's easier, and more pythonic. The class was not adding anything anyway.

### CODE REVIEW

**Comment by rwest** [comment]
@nateharms please could you look at this? a good way to get familiar with this bit of code...

File: rmgpy/qm/qmdata.py [hunk 1]
```diff
@@ -58,44 +54,36 @@ def __repr__(self):
...
         return 'QMData({0!s})'.format(string)
         
-class CCLibData(QMData):
+def parseCCLibData(cclibData, groundStateDegeneracy):
     """
-    QM Data extracted from a cclib data object
-    
-    This data objects collects information that CCLib was able to
-    retrieve from a quantum chemistry output file.
+    Parses a CCLib data object and returns CCLibData object.
```
Comment (rwest) [inline rmgpy/qm/qmdata.py]: It returns a QMData object

**Comment by rwest** [comment]
Looks fine to me (apart from the typo in the docstrings commented above).

**Comment by nickvandewiele** [comment]
thanks! Docstring is fixed.

**Pull request merged**